dalle_bot/dalle.py

```diff
@@ -49,6 +119,8 @@ def dalle(event, _):
         prompt = message['prompt']
         user = message['user']
 
+        prompt = sanitize_prompt(prompt, user)
```

Comment:
We should save off the original prompt to put into the response message so he doesn't notice we're manipulating his prompts